src/main/java/com/commercetools/http/converter/json/PrettyFormattedBody.java

```diff
@@ -0,0 +1,34 @@
+package com.commercetools.http.converter.json;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Class to wrap a response body if (JSON) pretty formatting could be expected in the result output.
```

Comment:
I am curious why you treat the `PrettyFormattedBody` that it **could be expected** to be pretty or not and always have to check inside of it if it's pretty or not? Why isn't it when `PrettyFormattedBody.of(tenantResponse)` is used, that its **for sure** that the response is pretty (also as the naming suggests) and otherwise use the regular not pretty response? And avoid caring about this checking logic in the `PrettyFormattedBody`.

Reply:
that's how i started, but then i landed to response entity create 

https://github.com/commercetools/commercetools-paypal-plus-integration/pull/100/files#diff-e1d923a05c4b52555104fead9e61dffaR53

and decided, that creating different body types based on request query parameter  even more confusing, than checking this property in the body writer implementation.
Also, this response body type could be reused by other controllers, so it will be also easier to use unified body factory with true-false property, rather than making `if-else` on every response returning.